pkg/monitoring/monitoring.go

```diff
@@ -218,7 +221,14 @@ func OverwriteAppAnswers(rawAnswers map[string]string, annotations map[string]st
 		rawAnswers[key] = value
 	}
 
-	return rawAnswers
+	return rawAnswers, GetMonitoringCatalogID(version)
+}
+
+func GetMonitoringCatalogID(version string) string {
+	if version != "" {
+		return fmt.Sprintf(cattleMonitoringPrefixCatalogID, version)
+	}
+	return settings.SystemMonitoringCatalogID.Get()
```

Comment:
I think the whole point of the issue is to get rid of this setting. But we are still using it here.

Reply:
updated to get the version from the defaultVersion` of catalog templates.